src/scss/reviews.scss

```diff
@@ -0,0 +1,127 @@
+$white-background: #ffffff;
+$main-background: #081130;
+
+.reviews__section {
+  background: $main-background;
+
+  padding: 80px 0;
+}
+
+.reviews__title {
+  margin-bottom: 50px;
+
+  color: #ffffff;
+  text-align: center;
+}
+
+.user-card {
+  display: flex;
+
+  padding: 70px 140px;
+
+  background-color: $white-background;
+  box-shadow: 2px 8px 16px rgba(194, 198, 208, 0.06);
+  border: 2px solid #E2EAF2;
```

Comment:
Color should be a variable.

Reply:
![image](https://user-images.githubusercontent.com/78299696/212350237-f2c099ae-fa50-4ce5-946b-288a940c3f66.png)